src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketCapabilitiesClientImpl.java

```diff
@@ -55,6 +55,10 @@ public BitbucketDeploymentCapabilities getDeploymentCapabilities() {
 
     @Override
     public AtlassianServerCapabilities getServerCapabilities() {
+        if (capabilitiesCache == null) {
+            capabilitiesCache = Suppliers.memoizeWithExpiration(new BitbucketCapabilitiesSupplier(bitbucketRequestExecutor),
```

Comment:
this is not safe, if you have more than one BbS instance defined you may get capabilities from the wrong one :( We need to think more about this.

Reply:
Following up, yes you're right we need a cache map here. Will also remove it from static context so it's a bit cleaner